language/diem-framework/transactional-tests/tests/prologue/fail_if_cant_pay_gas.move

```diff
@@ -0,0 +1,38 @@
+//# init --addresses Alice=0x76c3623ac80c38e2f1f4e90f07c8e21a
+
+
+//# run --signers 0xA550C18
+//#     --private-key 1b8c20cde2dbb43cd3c709b290ac50dcd2be2a87a3a24544b5a5109bc76ea7fb
```

Comment:
This looks really nasty. I had thought we would only migrate tests that don't need this at the moment, and then add private key management to the tests when we did need it. 

I'd rather add private key auto-management and land these particular tests later

Reply:
Well, the private key is mandatory for all transactions so there isn't really a test that doesn't need this. It shouldn't be hard for me to just implement private key management though.